app/pods/components/course-search-box/template.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="input-search nav-search">
-  {{input type="text" placeholder="Search Contents.." value=qs key-press=(perform searchTask) focusIn=(action 'showResult') focusOut=(action 'hideResult') }}
+  {{input type="text" placeholder="Search Contents.." value=qs key-press=(perform searchTask) focusIn=(action 'showResult') focusOut=(action 'hideResult') class="dsp-none-sm"}}
```

Comment:
class must be applied to parent. **TEST IT**

Reply:
If we add this to the parent it hides the input box but we only want to hide the placeholder text!